nautobot/docs/models/extras/relationship.md

```diff
@@ -54,36 +54,34 @@ The Circuit Relationship field will *not* show up on a Device with a role `leaf`
 
 ![Image of Leaf Device Relationships](../../img/leaf_dev_no_circuit_relationship.png)
 
-
-
 ## Relationship Labels
 
-Relationship connections can be labeled with a friendly name so that when they are displayed in the GUI, they will have a more descriptive or friendly name. 
-From the Devices/Circuits example above, you might label the relationship so that on the Device side the connection appears as 'Terminated Circuits' and on the Circuit side the connection appears as 'Terminating Devices'. 
+Relationship connections can be labeled with a friendly name so that when they are displayed in the GUI, they will have a more descriptive or friendly name.
+From the Devices/Circuits example above, you might label the relationship so that on the Device side the connection appears as 'Terminated Circuits' and on the Circuit side the connection appears as 'Terminating Devices'.
 
 ### Options
 
-It's also possible to hide the relationship from either side of the connection. 
+It's also possible to hide the relationship from either side of the connection.
 
-# Creating new Relationships
+## Creating New Relationships
 
 Relationships can be added through the UI under Extensibility > Relationships
 
-Each relationship must have a Name, Slug, Type, Source Object(s), and Destination Object(s). Optionally, Source Labels, Source Filters, Destination Labels, and Destination Filters may be configured. 
+Each relationship must have a Name, Slug, Type, Source Object(s), and Destination Object(s). Optionally, Source Labels, Source Filters, Destination Labels, and Destination Filters may be configured.
 
 Once a new relationship is added, the Relationship configuration section will appear under that device in the UI edit screen. Once a specific instance relationship has been configured for the object, that new relationship will appear under the Relationship section heading when viewing the object.
 
 When creating a relationship, if "Move to Advanced tab" is checked, this relationship won't appear on the object's main detail tab in the UI, but will appear in the "Advanced" tab. This is useful when the requirement is to hide this relationship from the main detail tab when, for instance, it is only required for machine-to-machine communication and not user consumption.
 
-# API
+## REST API
```

Comment:
Looks like the headings below at lines 82/80 and 98/96 should be changed from `##` to `###` since they're meant to be sub-headings under the `## REST API` heading?